src/cms/models/users/organization.py

```diff
@@ -3,6 +3,20 @@
 
 
 class Organization(models.Model):
+    """
+    Data model representing an organization
+
+    :param id: The database id of the organization
+    :param name: The name of the organization
+    :param slug: The slug of the organization
+    :param thumbnail: The url to the thumbnail image of the organization
+    :param created_date: The date and time when the organization was created
+    :param last_updated: The date and time when the organization was last updated
+
+    Reverse relationships:
+
+    :param members: All users who are members of organizations
```

Comment:
Just a small change proposal concerning the wording as it's about the members of a specific organization:
```suggestion
    :param members: All users who are members of the organization
```